roles/KubevirtCircuitBreaker/tasks/main.yml

```diff
@@ -0,0 +1,14 @@
+---
+# tasks file for KubevirtCircuitBreaker
+# 
+#
+- name: Extract the operator POD info
+  set_fact:
+    operator_pod: "{{ lookup('k8s', api_version='v1', kind='Pod', label_selector='name=kubevirt-ssp-operator') | from_yaml }}"
```

Comment:
I am a bit worried about the hardcoded label. Is there any way to introspect this?

Reply:
We need to check. But we need also to update the `KubevirtRepoInfo` role.

Reply:
Looks like there is no easy way to go up the chain CR -> deployment -> POD besides juggling with labels, which doesn't seem much of an improvement over the current CR.
On top of that, this is a bit hard to do in a generic way.
So if we fill it's a real concern we should tackle in a separate effort/PR

Reply:
Ack. The other option is to move the circuit breaker back to the CR. I wonder which direction is better.